llm-api-container fails to start when using docker compose yaml

When following the setup guide and using the default GPU compose file, I get an error with one of the containers llm-api-container  | TypeError: ForwardRef._evaluate() missing 1 required keyword-only argument: 'recursive_guard'
llm-api-container  | [2025-01-02 15:51:30 +0000] [8] [INFO] Worker exiting (pid: 8)
llm-api-container  | [2025-01-02 15:51:30 +0000] [1] [ERROR] Worker (pid:8) exited with code 3
llm-api-container  | [2025-01-02 15:51:30 +0000] [1] [ERROR] Shutting down: Master
llm-api-container  | [2025-01-02 15:51:30 +0000] [1] [ERROR] Reason: Worker failed to boot.

the other 2 containers boot up just fine but the api container does not. 

I tried forcing a python 3.11 environment for that container, but the same error occurs. 

Thanks for any help

--- a/dockerfile
+++ b/dockerfile
@@ -19,6 +19,8 @@ COPY . .
 
 COPY ./ragdir /ragdir
 
+RUN chmod +x use_model.sh
+
 ENTRYPOINT ["sh", "-c", "./use_model.sh && gunicorn app:app"]
 
 

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 llama-index==0.10.26
 llama-index-llms-ollama==0.1.2
-llama-index-embeddings-ollama==0.1.2
+llama-index-embeddings-ollama==0.1.3
 Flask==3.0.3
 Werkzeug==3.0.3
 redis==5.0.4

